web/source/dom/domEventHandlers.ts

```diff
@@ -361,8 +363,19 @@ namespace com.keyman.dom {
 
       DOMEventHandlers.states._SelectionControl = target;
 
+      if(target && outputTarget) {
+        // Call the current keyboard's newContext handler;
+        // timeout is required in order to get the current
+        // selection, which is not ready at time of focus event,
+        // at least on Chrome
+        window.setTimeout(() => {
+          //console.log('processNewContextEvent called from focus');
+          com.keyman.singleton.core.processNewContextEvent(outputTarget);
```

Comment:
Is this the same as? (`keyman` from L333)
```suggestion
          keyman.core.processNewContextEvent(outputTarget);
```

Reply:
ah, maybe it's not in the scope of setTimeout()

Reply:
It should be in scope and should work.